src/main/java/frc/robot/utils/debugging/SysIDCharacterization.java

```diff
@@ -0,0 +1,127 @@
+package frc.robot.utils.debugging;
+
+import com.ctre.phoenix6.SignalLogger;
+
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+
+// import static frc.robot.subsystems.drive.DriveConstants.kDrivebaseRadiusMeters;
+// import static frc.robot.subsystems.drive.DriveConstants.kRadiusMeters;
+
+import java.util.function.Consumer;
+
+import org.littletonrobotics.junction.Logger;
+
+// For rev logs extract using wpilib's data log tool:
+// https://docs.wpilib.org/en/stable/docs/software/telemetry/datalog-download.html
+// For talon logs extract using phoenix tuner x:
+// https://pro.docs.ctr-electronics.com/en/latest/docs/tuner/tools/log-extractor.html
+public class SysIDCharacterization {
+    public static Command runShooterSysIDTests(Consumer<Double> voltageSetter, Subsystem subsystem) {
```

Comment:
This is shooter specific, is it not? @AnshulBompada

Reply:
Yes, but it works for all "flywheel" systems, so I opted it best to keep it there. I will rename it to runFlywheelSysIDTests to make things more clear.